client/cl_property.lua

```diff
@@ -1,3 +1,5 @@
+currentPropertyId = nil
```

Comment:
@ImXirvin Do you have any fancier way of exposing the currentPropertyId to an export? This was my suggestion to Lesiin as a quick way to get it